File: ProgrammingFundamentals/Java/04.MethodsDebuggingTroubleshootingCode/Lab/src/P02_SignOfIntegerNumber.java
import java.util.Scanner;
public class P02_SignOfIntegerNumber {
    public static void main(String[] args) {
        Scanner console = new Scanner(System.in);
        int n = Integer.parseInt(console.nextLine());
        printSign(n);
    }

    static void printSign(int number) {
        if(number > 0) {
            System.out.format("The number %d is positive.", number);
        } else if(number < 0) {
            System.out.format("The number %d is negative.", number);
        } else {
            System.out.format("The number %d is zero.", number);
        }
    }
}

File: ProgrammingFundamentals/C#/04.MethodsDebuggingAndTroubleshootingCode/Lab/P02_SignOFIntegerNumber/P02_SignOFIntegerNumber.cs
using System;

namespace P02_SignOFIntegerNumber
{
    class P02_SignOFIntegerNumber
    {
        static void Main(string[] args)
        {
            int n = int.Parse(Console.ReadLine());
            PrintSign(n);
        }

        static void PrintSign(int num)
        {
            if(num > 0)
            {
                Console.WriteLine($"The number {num} is positive.");
            }
            else if(num == 0)
            {
                Console.WriteLine($"The number {num} is zero.");
            }
            else
            {
                Console.WriteLine($"The number {num} is negative.");
            }
        }
    }
}
